### PR TITLE
Ria 3877 ooc

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3877

### Change description ###
See ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: e2e/features/start-appeal-out-of-country-removal-of-client.feature [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Feature: Out of country appeal removal of client
```
Comment (Lee365) [inline e2e/features/start-appeal-out-of-country-removal-of-client.feature:1]: "I save my initial appeal" journey to include ooc journey with check on IA_OUT_OF_COUNTRY_ENABLED

**Review by Lee365 — CHANGES_REQUESTED** [review]

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by Lee365 — APPROVED** [review]
LGTM

**Pull request merged**